src/sentry/backup/imports.py

```diff
@@ -49,6 +52,23 @@ def __init__(self, context: RpcImportError) -> None:
         self.context = context
 
 
+def reset_models():
```

Comment:
```suggestion
def _clear_model_tables_before_import():
```

More clearly describes what this method does, and when we expect it to be used. Also makes clear that it is not for external consumption.